civis/tests/test_io.py

```diff
@@ -972,35 +973,6 @@ def test_check_column_types_passing(self, _m_get_api_spec):
 
         civis.io._tables._check_column_types(table_cols, file_cols, 42)
 
-    @pytest.mark.read_civis
-    @pytest.mark.skipif(not has_pandas, reason="pandas not installed")
-    @mock.patch(api_import_str, return_value=civis_api_spec)
-    def test_read_civis_pandas(self, *mocks):
```

Comment:
These cassette-based tests wouldn't pass. Since we've been gradually removing/replacing these cassette tests with mock-based ones anyway (e.g, #328, #425), I'm doing the same here with the `read_civis(_sql)` tests. (`read_civis` is a thin wrapper of `read_civis_sql`, and so below I'm explicitly testing `read_civis_sql` only.)